packages/xo-vmdk-to-vhd/package.json

```diff
@@ -28,7 +28,6 @@
     "child-process-promise": "^2.0.3",
     "core-js": "^3.0.0",
     "lodash": "^4.17.15",
-    "pipette": "^0.9.3",
```

Comment:
Why `yarn.lock` wasn't updated?

Reply:
I don't compile on the same machine where I develop. I'll update it.